client/src/components/Navbar/Navbar.js

```diff
@@ -4,7 +4,8 @@ class Navbar extends Component {
   render() {
     return (
       <div className="navbar">
-        Navbar Component
+        <img className="navbar-comrad-logo" src="https://i.postimg.cc/pdBJd3L2/KGNUwhite.png"/>
```

Comment:
@dzimmerman902 I am not sure how to reference a local image can you help me with that?

Reply:
btw this pull request is not for review, I am adding a drop down under the user icon

Reply:
You can save your image inside the images file. Then you can import the file using a relative path inside the component you want to use it in.